Dockerfile

```diff
@@ -3,4 +3,5 @@ FROM python:2.7-onbuild
 RUN apt-get update \
     && apt-get install -y python-protobuf
```

Comment:
I added this to the Dockerfile because Protobuf 3 library is declared as an installation requirement, see Wiki: [https://github.com/PokemonGoF/PokemonGo-Bot/wiki/Installation#requirements-click-each-one-for-install-guide](https://github.com/PokemonGoF/PokemonGo-Bot/wiki/Installation#requirements-click-each-one-for-install-guide)